PracticeGit/Complex.h

```diff
@@ -0,0 +1,30 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------------------------------------
+// Complex.h - Complex numbers 
+// ------------------------------------------------------------------------------------------------
+
+#ifndef COMPLEX_H
+#define COMPLEX_H
+
+typedef struct {
+   float real, img;

```

Comment:
```suggestion
   float Real, Img;

```
Reviews given previously are valid for all future assignments as well.